src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+const BOOKADDED = 'redux/books/addBook';
+const BOOKREMOVED = 'redux/books/removeBook';
+
+const initialState = {
+  books: [],
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case BOOKADDED:
+      return [
+        ...state,
+      ];
+    case BOOKREMOVED:
+      return [
+        ...state,
+      ];
```

Comment:
- [ ] Kindly make sure that the `BOOKREMOVED` changes the state and filters out the removed book. You can use the [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) array method to filter out the removed book.